bitcoin/src/blockdata/fee_rate.rs

```diff
@@ -95,13 +97,14 @@ impl FeeRate {
     ///
     /// # Examples
     ///
-    /// ```no_run
```

Comment:
Why was this `no_run`?

Reply:
I'm not sure, but I think it was because the example didn't assert or panic, so running it would be pointless. Now that we have an `assert_eq` call it makes sense to run it.